templates/index

```diff
@@ -132,10 +132,12 @@ cisco_asa_show_version.textfsm, .*, cisco_asa, sh[[ow]] ver[[sion]]
 cisco_asa_show_route.textfsm, .*, cisco_asa, sh[[ow]] ro[[ute]]
 cisco_asa_show_xlate.textfsm, .*, cisco_asa, sh[[ow]] x[[late]]
 cisco_asa_show_name.textfsm, .*, cisco_asa, sh[[ow]] nam[[e]]
-cisco_asa_show_arp.textfsm, .*, cisco_asa, sh[[ow]] arp
+cisco_asa_show_arp.textfsm, .*, cisco_(asa|ftd), sh[[ow]] arp
 cisco_asa_show_nat.textfsm, .*, cisco_asa, sh[[ow]] nat
 cisco_asa_dir.textfsm, .*, cisco_asa, dir
 
+cisco_ftd_ping.textfsm, .*, cisco_ftd, ping
```

Comment:
Do you mind just adding this command within ASA as well? And then using the same logic you did for `show_arp`?

Reply:
Updated the line so it works for both platforms.